lib/raxx/simple_server.ex

```diff
@@ -0,0 +1,111 @@
+defmodule Raxx.SimpleServer do
+  @moduledoc """
+  Server interface for simple `request -> response` interactions.
+
+  ## Example
+
+  Echo the body of a request to the client
+
+      defmodule EchoServer do
+        use Raxx.SimpleServer, maximum_body_length: 12 * 1024 * 1024
```

Comment:
I think it's worth pointing out that `use Raxx.SimpleServer` results in a module that does implement `Raxx.Server`, as the latter is the de-facto interface that the implementations care about, but it's perfectly fine if it just happens under the hood and you don't have to worry about it.

Probably needs a better phrasing than the one I provided :D